tests/crypto/sse/findex/implementations/postgrest/tests/findex.test.ts

```diff
@@ -23,6 +23,11 @@ test("upsert+search", async () => {
   expect(entries.length).toBe(577);
   expect(chains.length).toBe(792);
 
+  const progress = (res: Uint8Array[]) => {
+    console.log(res);
```

Comment:
Maybe add a test to check if the `progress` callback receive all the element by incrementing a counter and comparing with the final result?